src/runtime/parser.c

```diff
@@ -87,7 +87,7 @@ typedef enum {
 } ErrorComparison;
 
 typedef struct {
-  const char *string;
+  const void *string;
```

Comment:
I think that this field should stay as a `char` pointer, rather than a `void` pointer. Code below this in `ts_string_input_read` expects to be able to do byte-wise pointer arithmetic on the field. It looks like that's why the windows CI is failing.